core/server/common/src/main/java/alluxio/RpcUtils.java

```diff
@@ -39,10 +39,35 @@
     try {
       return callable.call();
     } catch (AlluxioException e) {
-      LOG.debug("Internal Alluxio error when running rpc", e);
+      LOG.debug("Internal Error: {}", callable, e);
```

Comment:
This case is unique in that it's the last place where we see the exception before it's sent over the wire. Client logs can be harder to track down, so at debug level I think it's reasonable to log and throw here.